parser/src/grammar/Python.asdl

```diff
@@ -9,7 +9,6 @@ module Python
     mod = Module(stmt* body)
         | Interactive(stmt* body)
         | Expression(expr body)
-
```

Comment:
why remove the whitespace here? just a formatting change?

Reply:
Not sure, it wasn't intentional.

Reply:
if it is extra then we can remove it -- if it was there to provide some visual segregation then may be worth keeping in!

Reply:
Yeah, I'll undo this change and make adjustments based on your other comments.

Also, I'll add some better documentation to this library.